stix_shifter_modules/rhacs/README.md

```diff
@@ -0,0 +1,603 @@
+# RHACS(StackRox) Connector
+
+**Table of Contents**
+
+- [RHACS API Endpoints](#rhacs-api-endpoints)
+- [Pattern expression with STIX attributes - Single Observation](#single-observation)
+- [Pattern expression with STIX attributes - Multiple Observation](#multiple-observation)
+- [Pattern expression with STIX attributes - Execute Query](#stix-execute-query)
+- [Limitations](#limitations)
+- [References](#references)
+
+### RHACS API Endpoints
+
+   |Connector Method|RHACS API Endpoint| Method
+   | ----           |   ------              | -----|
+   |Alerts Endpoint  |https://<{fqdn}>v1/alerts?query=xxxx|GET
+   |Alert by id Endpoint|https://<{fqdn}>v1/alerts/alertid|GET
+   |Ping Endpoint|https://<{fqdn}>v1/ping|GET
+   
+### Format for calling stix-shifter from the command line
+```
+python main.py `<translate or transmit>` `<translator_module>` `<query or result>` `<STIX identity object>` `<data>`
+```
+### Pattern expression with STIX attributes
+
+### Single Observation
+
+#### STIX Translate query
+```shell
+translate rhacs query '{}' "[x-rhacs-cluster:name = 'xxxx'] START t'2022-07-07T11:00:00.000Z' STOP t'2022-07-08T11:00:00.003Z'"
+```
+#### STIX Translate query - output
+```json
+{
+    "queries": [
+        "Cluster:\"xxxx\"%2BViolation Time:>=07/07/2022"
```

Comment:
is there any reason why query translation totally disregard the hr/mins/seconds from the time ranges?